droid-export/src/main/java/uk/gov/nationalarchives/droid/export/template/ExportTemplateBuilder.java

```diff
@@ -192,7 +194,7 @@ private ProfileResourceNodeColumnDef createProfileNodeDef(String header, String
             throw new ExportTemplateParseException(String.format(messageFormat, param2));
         }
         String originalColumnName = param2.substring(1);
-        if (!(Arrays.stream(WriterConstants.HEADERS).collect(Collectors.toList()).contains(originalColumnName))) {
+        if (!(Arrays.stream(WriterConstants.HEADERS).toList().contains(originalColumnName))) {
```

Comment:
I think you can do `Arrays.asList(WriterConstants.HEADERS)` It's slightly shorter.